pkg/virt-controller/watch/vmi.go

```diff
@@ -729,6 +745,17 @@ func (c *VMIController) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod,
 		if !vmi.IsUnprocessed() {
 			return nil
 		}
+		// let's check if we already have topology hints or if we are still waiting for them
+		if vmi.Status.TopologyHints == nil {
+			if topologyHints, err := c.topologyHinter.TopologyHintsForVMI(vmi); err != nil {
+				c.recorder.Eventf(vmi, k8sv1.EventTypeWarning, FailedGatherhingClusterTopologyHints, err.Error())
```

Comment:
this event is probably getting fired twice on every reconcile loop for a VMI when topology hints can't be found. 

maybe we could keep this